### DISCUSSION TITLE
Screen borders

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#643] [Q&A] [answered]
I had just an update of the Realme UI and since then, the screen borders of fullscreen mode have "changed". In the settings all borders are set to zero but the screen looks as follows:
![Screenshot_2024-10-16-11-28-01-11_beed72cb86d752c6022398a83c94aee3](https://github.com/user-attachments/assets/5b43a295-37cd-4c23-b543-5c5674f261c1)

Now there is a margin probably because of the camera hole top left of the screen.
Changing to landscape looks as follows (camera hole is now left bottom on the screen):
![Screenshot_2024-10-16-13-18-20-39_beed72cb86d752c6022398a83c94aee3](https://github.com/user-attachments/assets/c46f5510-483e-4b86-93e3-222445141b5d)

Probably I need now negative values for the screen borders?

**Comment by kds112** [comment]
Turning the landscape view by 180degrees looks as follows (camera hole is top right) :
![Screenshot_2024-10-16-13-30-32-62_beed72cb86d752c6022398a83c94aee3](https://github.com/user-attachments/assets/1e4cabbe-f0df-4299-98f5-d3ac76b6523f)

**Comment by devemux86** [comment]
@kds112 Thanks for the report. How it worked before?

Devices with camera holes or cuts usually have system settings to adjust them.
We cannot place the items "around" the holes / cuts, only at their edges.
Imagine the top panel in portrait navigation, it would be below the hole and cannot be seen.

**Comment by kds112** [comment]
Before the compass top left was partly behind the camera hole. Now it looks like that the original of coordinates has moved by the system.

**Comment by devemux86** [comment]
I have an idea, we can try something in the next update and see if it works.

**Comment by devemux86** [comment]
Please try with **5.0.1-b3**.

**Reply by kds112** [reply, marked as answer]
B3 has solved the effect👍 Thanks a lot🏁

**Reply by devemux86** [reply]
When navigation is active and the top panels are enabled, aren't they behind the hole / cutout?

@kds112 Can you show a screenshot of your device with navigation + top panels?
Screenshot where we can see the camera hole.

**Reply by kds112** [reply]
Here it is... I marked the camera hole top left
![Uploading IMG_20241016_165008.jpg…]()

**Reply by devemux86** [reply]
@kds112 The screenshot was not uploaded.

**Reply by kds112** [reply]
![Uploading IMG_20241016_165008.jpg…]()

**Reply by devemux86** [reply]
@kds112 Same, must wait the upload to finish and then post the comment.
Can also check the "Preview" of the answer to see if everything is correct.

**Reply by kds112** [reply]
![IMG_20241016_165008](https://github.com/user-attachments/assets/2ec06575-66d5-4576-99b9-b965b75e55f1)

**Reply by devemux86** [reply]
@kds112 Thanks!

This could cause problems, as other devices have it in the top center or have (large) notches.
Although they usually also have specific settings on how apps should be located under or around these gaps.

**Comment by MotoUKRider** [comment]
For info, v5.0.1 and b3+b4 show no difference on my Samsung S20 whether in "Browse" , "Follow" or "Navigation" in either orientation.
